src/main/java/org/jenkinsci/plugins/DependencyCheck/DependencyTrackPublisher.java

```diff
@@ -61,7 +61,7 @@
 
     private final String projectId;
     private final String scanResult;
-    private PrintStream logger;
+    private transient PrintStream logger;
```

Comment:
@stevespringett: I don't know enough about Jenkins to see the consequences of this change, but I guess we cannot have a non-serializable field here.